src/drivers/postgres/schema.js

```diff
@@ -67,6 +69,7 @@ module.exports = {
         // https://www.postgresql.org/docs/9.5/infoschema-table-constraints.html
         primaryKey,
         unique: row.constraint_type === 'UNIQUE',
+        datetimePrecision: precision === 6 ? null : precision,
```

Comment:
postgres has datetime_precision default to 6 even if the precision is not specified in column definition. reset it back to `null` to work around existing test cases for now.